help/stats/2021_by_area/index.html

```diff
@@ -0,0 +1,358 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
This is the file originally supplied by PG; it has been translated to markdown with index.html.